retriever/lib/templates.py

```diff
@@ -1,24 +1,28 @@
 """Class models for dataset scripts from various locations. Scripts should
 inherit from the most specific class available."""
 from __future__ import print_function
-from builtins import object
+
 import os
 import shutil
-from retriever.lib.defaults import DATA_DIR
+from builtins import object
+
 from retriever.lib.models import *
 from retriever.engines import choose_engine
+from retriever.lib.defaults import DATA_DIR
 
 
 class Script(object):
-    """This class represents a database toolkit script. Scripts should inherit
-    from this class and execute their code in the download method."""
+    """This class represents a database toolkit script.
```

Comment:
Since we're in here cleaning up let's remote "toolkit" which dates to the early early days where this project was called "Database toolkit".